bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_VBF_mNgtmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_VBF_V-0p01_e_NLO_run_card.dat

```diff
@@ -0,0 +1,167 @@
+  2.0	=  rw_Rscale_up      ! upper bound for ren scale variations
+  0.5	=  rw_Fscale_down    ! lower bound for fact scale variations
+  2.0	=  rw_Fscale_up      ! upper bound for fact scale variations
+  True	=  reweight_PDF      ! reweight to get PDF uncertainty
+#***********************************************************************
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+ True = store_rwgt_info ! Store info for reweighting in LHE file     
+#***********************************************************************
+# ickkw parameter:                                                     *
+#   0: No merging                                                      *
+#   3: FxFx Merging - WARNING! Applies merging only at the hard-event  *
+#      level. After showering an MLM-type merging should be applied as *
+#      well. See http://amcatnlo.cern.ch/FxFx_merging.htm for details. *
+#   4: UNLOPS merging (with pythia8 only). No interface from within    *
+#      MG5_aMC available, but available in Pythia8.                    *
+#  -1: NNLL+NLO jet-veto computation. See arxiv:1412.8408 [hep-ph].    *
+#***********************************************************************
+  0	=  ickkw
+ #***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+  15.0	=  bwcutoff
+ #***********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+  1.0	=  jetalgo    ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+  0.7	=  jetradius  ! The radius parameter for the jet algorithm
+  20.0	=  ptj        ! Min jet transverse momentum
+  -1.0	=  etaj       ! Max jet abs(pseudo-rap) (a value .lt.0 means no cut)
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+  0.0	=  ptl      ! Min lepton transverse momentum
+  10.0	=  etal     ! Max lepton abs(pseudo-rap) (a value .lt.0 means no cut)
```

Comment:
Did you synchronise your VBF settings to Richard Ruiz's paper? Doesn't look like so... This will create some ill behaviour to your samples

Reply:
I inverted the eta and pt conditions, thanks for catching this. I have now realigned the pt and eta cuts with the paper.